app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -55,6 +55,7 @@ open class FenixApplication : Application() {
 
         setupMegazord()
         setupLogging()
+        setupPush()
```

Comment:
I assume this needs to move lower and should only happen in the main process and not all processes.